releases/zeppelin-release-0.7.1.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Apache Zeppelin Release 0.7.1"
+description: ""
+group: release
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+## Apache Zeppelin Release 0.7.1
+
+The Apache Zeppelin community is pleased to announce the availability of the 0.7.1 release.
+
+The community put significant effort into improving Apache Zeppelin since the last release focusing on restarting interpreter process stability, python interpreter improvement, table/chart rendering bug fix.
+24 contributors provided 80+ patches for improvements and bug fixes.
+More than 70+ issues have been resolved.
+
+We encourage to [download](../../download.html) the latest release. Feedback through the [mailing lists](../../community.html) is very welcome.
+
+### Improvements
+   * Rewrite python interpreter for streaming output and seamless matplotlib support
+   * Make dynamic select form turn on or off using checkbox
```

Comment:
@felixcheung You mean it's not intuitive, right? Currently there is no way to let ppl know they can turn it on/off. Let me improve it in the near future :)